include/snowflake/platform.h

```diff
@@ -46,7 +46,7 @@ typedef pthread_mutex_t SF_MUTEX_HANDLE;
 
 #endif
 
-#include "Simba_CRTFunctionSafe.h"
+#include "SF_CRTFunctionSafe.h"
```

Comment:
to be compatible we should still include here `Simba_CRTFunctionSafe.h` because when include only `platform.h` then `sb_*` operations are missing?